### DISCUSSION TITLE
Any way to get VSC status bar color change per mode?

### DISCUSSION THREAD

**Discussion by proofer** [vscode-neovim/vscode-neovim#1021] [Q&A] [answered]
I just switched from VSCodeVim and I miss having the VSC status bar color change when vim's mode changes (Normal, Insert, etc.) Is there  *currently* a way to get this in VSCode Neovim? If not, I'd like to see it in a future version.

**Comment by theol0403** [comment, marked as answer]
Not currently a way to set vscode settings from neovim without 3rd party extension that exposes a command. 
You can do this with settings cycler and other similar plugins.

There is also this which might do what you want https://marketplace.visualstudio.com/items?itemName=JulianIaquinandi.nvim-ui-modifier

**Reply by proofer** [reply]
I don't think the extension you linked does what I want.  I've been looking at code, but I'm not optimistic -- I'm in over my head. So for now I'll just reiterate: I'd like to see it in a future version. Thanks!

(And while I"m here -- thanks for maintaining VSCode-NeoVim!)

**Comment by wrath-codes** [comment]
I wrote an extension for that, if it helps at all:
[Nvim UI+](https://marketplace.visualstudio.com/items?itemName=wrathcodes.nvim-ui-plus)
